controller/network/path.go

```diff
@@ -26,10 +26,11 @@ import (
 )
 
 type Path struct {
-	Nodes     []*Router
-	Links     []*Link
-	IngressId string
-	EgressId  string
+	Nodes               []*Router
+	Links               []*Link
+	IngressId           string
+	EgressId            string
+	TerminatorLocalAddr string
 }
 
```

Comment:
It looks like the formatting is off a bit. Can you check if you're running gofmt on save?

Reply:
I am running format on save. What is the expected formatting here? It looks like I would have expected it, but maybe I'm expecting something different.